changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## `v0.16.0`
+
+### Features:
+
+- Added ability to pass a genesis file and peer address to `starport network chain join`
+- Added ability to show genesis or peer list with a flag in `starport network chain show`
+- `protoc` is now bundled with Starport CLI
+- Starport is now published automatically on the Docker Hub
+- Added Keplr extension support to the web app
```

Comment:
```suggestion
- Keplr extensions are supported on web apps created with Starpot
```

Reply:
Changed "extensions" to "extension" (it's a single thing) and "Starpot" to "Starport".